src/components/Badge.vue

```diff
@@ -53,20 +54,18 @@ export default {
 
   @include font-styles(badge);
   display: inline-block;
-  padding: 2px 10px;
+  padding: 2px 4px;
   white-space: nowrap;
-  background: none;
-  border-radius: var(--badge-border-radius, calc(#{$border-radius} - 1px));
-  border-style: var(--badge-border-style, solid);
```

Comment:
Should we keep these and just change the defaults so that users can continue to theme the style/width of the badge border?

Reply:
Good point!